Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.13 as builder
+
+COPY . /go/src/github.com/mrtazz/checkmake 
+
+RUN cd /go/src/github.com/mrtazz/checkmake && GOOS=linux GOARCH=amd64 CGO_ENABLED=0 make binaries
+
+FROM alpine:3.9
+
+USER nobody
+
+COPY --from=builder /go/src/github.com/mrtazz/checkmake/checkmake /
```

Comment:
Add 
```
WORKDIR /data
ENTRYPOINT /checkmake
CMD -h
```
So that the user can execute it more easily:
```
docker run --rm docker.pkg.github.com/mrtazz/checkmake/checkmake:f127301db6ce6d9861ea0c9b94cd08a9d023fb77
#or
docker run --rm docker.pkg.github.com/mrtazz/checkmake/checkmake:f127301db6ce6d9861ea0c9b94cd08a9d023fb77 --version
#or 
docker run --rm -v $(pwd):/data docker.pkg.github.com/mrtazz/checkmake/checkmake:f127301db6ce6d9861ea0c9b94cd08a9d023fb77 makefile
```

I use a wrapper to use this dockerized tool locally the same way i would run it with a normal installed program (and at the same time keep my local system clean):
```
checkmake makefile
```
More inspiration:
https://hub.docker.com/r/freifunkhamm/mapshaper
https://hub.docker.com/r/freifunkhamm/jsonschema/

Please consider to upload that docker image to docker hub instead. Docker images at docker hub are more easy to access for users, as it do not force the user to create a personal access token (and to grant "repo" and "read:packages" permissions) at github just to be able to pull it.